listenbrainz/websockets/webserver.py

```diff
@@ -0,0 +1,60 @@
+import eventlet
+eventlet.monkey_patch()
+
+from flask_socketio import SocketIO, join_room, leave_room, emit
+from werkzeug.exceptions import BadRequest
+
+from listenbrainz.webserver import load_config
+from brainzutils.flask import CustomFlask
+
+app = CustomFlask(
+    import_name=__name__,
+    use_flask_uuid=True,
+)
+load_config(app)
+
+# Error handling
+from listenbrainz.webserver.errors import init_error_handlers
+init_error_handlers(app)
+
+# Logging
+app.init_loggers(
+    file_config=app.config.get('LOG_FILE'),
+    email_config=app.config.get('LOG_EMAIL'),
+    sentry_config=app.config.get('LOG_SENTRY')
+)
+socketio = SocketIO(app, cors_allowed_origins='*')
+
+
+def get_playlist_id(json):
+    identifier = json['identifier']
+    idx = identifier.rfind('/')
+    return identifier[idx + 1:]
+
+
+@socketio.on('change_playlist')
+def dispatch_playlist_updates(data):
+    emit('playlist_changed', data, room=get_playlist_id(data))
+
+
+@socketio.on('joined')
+def joined(data):
```

Comment:
Should we also notify the other users in cases like the collab playlists?

Reply:
That seems to be a different feature (sounds more like notifications). Here, we are just showing updates to anyone who has the playlist opened in their browser.

Reply:
Makes sense